cirq/ops/common_gates.py

```diff
@@ -982,3 +982,82 @@ def Ry(rads: float) -> RotYGate:
 def Rz(rads: float) -> RotZGate:
     """Returns a gate with the matrix e^{-i Z rads / 2}."""
     return RotZGate(rads=rads, global_shift_in_half_turns=-0.5)
+
+
+class XXGate(eigen_gate.EigenGate,
+                gate_features.TwoQubitGate,
+                gate_features.InterchangeableQubitsGate):
+    """XX interaction.
+
+    The gate implements the following unitary:
+    exp(-i t X⊗X) = [cos(t) 0 0 -isin(t)]
+                    [0 cos(t) -isin(t) 0]
+                    [0 -isin(t) cos(t) 0]
+                    [-isin(t) 0 0 cos(t)]
+
+    """
+
+    def __init__(self, *,  # Forces keyword args.
+                 duration: Optional[float]=None,
```

Comment:
This duration concept is sort of unique to openfermion-cirq and has no equivalent in cirq, so we should leave it out. If we want a gate specified with alternate units, we'll make a utility method analogous to `Rx`.

Is there a typical name for a function that takes radians and spits out exp(i XX t)?